Quantum-Control-Applications/Superconducting/Multiple Flux Tunable Transmons/Use Case 2 - Two Qubit Randomized Benchmarking/README.md

```diff
@@ -0,0 +1,238 @@
+
+```python
+def bake_cz(baker: Baking, q0, q1):
+    q0_xy_element = f"qubit{q0}_xy"
+    q1_xy_element = f"qubit{q1}_xy"
+    q1_z_element = f"qubit{q1}_z"
+    baker.play("cz", q1_z_element)
+    baker.align()
+    baker.frame_rotation_2pi(qubit0_frame_update, q0_xy_element)
+    baker.frame_rotation_2pi(qubit1_frame_update, q1_xy_element)
+    baker.align()
+```
+
+## Initialization
+Before each circuit, it is important to implement a initialization protocol to reset the qubits to the ground state. In the example the *prep* function contains a single QUA command *wait* and is called before each circuit execution to assure that the initial state is set to |00>. The time inside the *wait* statement is chosen to be a multiple of the characteristic decay time of the qubits *T1* to leave enough time for the qubit to relax after it has been excited to the excited state |1>. If single shot readout is implemented, it is possible to use active feedback to reset the qubit to the ground state |0> by sending a pi-pulse if the qubit was measured in the excited state |1>.   
+
+```python
+T1 = 10000
+def prep():
+    wait(10*T1)  # thermal preparation
+    align()
+```
+
+## Measurement
+Finally, the user has to implement a measurement that is performed at the end of the random gate circuits. In this example we send a readout pulse to the resonators of the qubits and demodulate the signal, where we assume that we optimized the readout such that all information is contained in the *Iq0* and *Iq1*. We then assign a True or False value to boolean QUA variables *state0* and *state1* and return the result, where False should be returned for state |0> and True for state |1>. This the measurement function returns four possible outcomes: |00>, |01>, |10> and |11>
+
+```python
+def meas():
+    threshold0 = 0.3 #example value
+    threshold1 = 0.3 #example value
+    rr0_name = f"qubit{q0}_rr"
+    rr1_name = f"qubit{q1}_rr"
+    Iq0 = declare(fixed)
+    Qq0 = declare(fixed)
+    Iq1 = declare(fixed)
+    Qq1 = declare(fixed)
+    state0 = declare(bool)
+    state1 = declare(bool)
+    measure("readout", rr0_name, None,
+            dual_demod.full("w1", "out1", "w2", "out2", Iq0),
+            dual_demod.full("w3", "out1", "w1", "out2", Qq0)
+            )
+    measure("readout", rr1_name, None,
+            dual_demod.full("w1", "out1", "w2", "out2", Iq1),
+            dual_demod.full("w3", "out1", "w1", "out2", Qq1)
+            )
```

Comment:
Here you are reading the 2 resonators simultaneously. I think it is important to emphasize it because the optimal readout parameters will differ if they were optimized for sequential readout

Reply:
I am not sure this is related to the 2 qubit RB, but up to you guys